superset/views/base.py

```diff
@@ -359,19 +361,21 @@ def yaml_export(self, items):
 
 
 class DeleteMixin:  # pylint: disable=too-few-public-methods
-    def _delete(self, primary_key):
+    def _delete(
+        self: Union[BaseView, "DeleteMixin", DruidClusterModelView], primary_key: int,
+    ) -> None:
         """
             Delete function logic, override to implement diferent logic
             deletes the record with primary_key = primary_key
 
             :param primary_key:
                 record primary key to delete
         """
-        item = self.datamodel.get(primary_key, self._base_filters)
+        item = self.datamodel.get(primary_key, self._base_filters)  # type: ignore
```

Comment:
Because FAB isn't typed and there's no stubs it seems simpler to just ignore these.